application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +53,22 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(res -> {
+                logger.info("This is a Error handler of DM");
+            }, new ErrorHandler().handle(ErrorResponse.CANNOT_SEND_TO_USER, err -> {
```

Comment:
u don't need to do all that
```java
message.delete().flatMap(any-> dmUser(message)).queue(any -> {}, err-> { ... });
```
just do it like this

Reply:
sure this sounds much simpler